usr/bin/irdata_text_pad.py

```diff
@@ -38,38 +33,25 @@ def main():
     progname = os.path.basename(sys.argv[0])
 
     # Get the command line options.
+    argparser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    argparser.add_argument("-n", metavar="fields", required=True, type=int)
+    argparser.add_argument("-p", metavar="padding", required=True)
```

Comment:
My change here probably breaks formatting (suggest you don't use the "commit suggestion" button)!

```suggestion
    argparser.add_argument("-n", metavar="fields", required=True, type=int, dest="fields")
    argparser.add_argument("-p", metavar="padding", required=True, dest="padding")
```